src/plugin/PluginManager.php

```diff
@@ -77,6 +78,9 @@ class PluginManager{
 	 */
 	protected $enabledPlugins = [];
 
+	protected $scanLoaded = false;
+	protected $scanEnabled = false;
```

Comment:
please add `@var` annotations to these fields

Reply:
The name `scanEnabled` is ambiguous. I initially took it to mean "scanning is enabled", but it seems to actually mean "scan load-order plugins have been enabled".

Reply:
`@var` annotations for types or for actual documentation? It's a private field (it's protected, but effectively private).

What other name would you expect?

Reply:
For the types. All fields should have type annotations, regardless of visibility.